<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISideScreen</class>
 <widget class="QWidget" name="UISideScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UISideScreen</string>
  </property>
  <widget class="QPushButton" name="pack_pushBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>隐藏副屏窗口</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/LoginWindow/images/camera_pack.png)};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tip_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头开启中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="swichScreen_pushBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>切换屏幕</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/LoginWindow/images/switch.png)};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loginwindow.qrc"/>
 </resources>
 <connections/>
</ui>
